scripts/make-goldens.js

```diff
@@ -20,20 +20,23 @@
  */
 
 const fs = require('fs');
+const fsExtra = require('fs-extra');
 const path = require('path');
-const rimraf = require('rimraf');
+
 const {generateDeclarations} = require('../lib/gen-ts');
 
 const fixturesDir = path.join(__dirname, '..', 'src', 'test', 'fixtures');
 const goldensDir = path.join(__dirname, '..', 'src', 'test', 'goldens');
 
-rimraf.sync(goldensDir);
-fs.mkdirSync(goldensDir);
+fsExtra.emptyDir(goldensDir);
```

Comment:
Why this change?

Reply:
I added a dependency on `fs-extra` because I wanted `mkdirs` and `outputFile`. Since `fs-extra` includes `emptyDir`, which is equivalent to `rimraf` followed by `fs.mkdir`, I used that instead so that I could drop the dependency on `rimraf`.